app/views/csv_preview/show.html.erb

```diff
@@ -1,70 +1,68 @@
-  <div class="organisation-logos">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full govuk-!-margin-bottom-3 govuk-!-margin-top-6">
     <%- @edition.organisations.each do |organisation| %>
-      <div class="organisation-logos__logo <%= organisation_brand_colour_class(organisation) %>">
-        <%= link_to organisation_path(organisation),
-                    class: logo_classes(organisation: organisation, size: 'medium', stacked: true) do %>
-          <span><%= organisation_logo_name(organisation) %></span>
-        <%- end -%>
-      </div>
+      <%= render "govuk_publishing_components/components/organisation_logo", {
+        organisation: {
+          name: organisation_logo_name(organisation),
+          url: organisation_path(organisation),
+          brand: organisation.organisation_brand_colour ? organisation.organisation_brand_colour.class_name : nil,
+          crest: organisation.organisation_logo_type.class_name,
+        }
+      } %>
     <%- end -%>
   </div>
-  <div>
-    <p><%= link_to "See more information about #{with_this_determiner(@edition.display_type.downcase)}", public_document_url(@edition) %></p>
-  </div>
 </div>
-<header class="publication-header" id="contents">
-  <div class="inner-block floated-children">
-    <div class="headings">
-      <%= content_tag(:p, @edition.display_type, class: 'document-type') %>
-      <h1><%= @attachment.title %></h1>
-      <p>
-        <%= published_or_updated(@edition).titlecase %> <%= absolute_date(@edition.public_timestamp, class: 'published-at') %><br />
-        <%= link_to @attachment.url(preview: params[:preview]) do %>
-          <strong>Download <%= @attachment.file_extension.upcase %></strong>
-          <%= number_to_human_size(@attachment.file_size) %>
-        <% end %>
-      </p>
-    </div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <p class="govuk-body"><%= link_to "See more information about #{with_this_determiner(@edition.display_type.downcase)}", public_document_url(@edition), class: "govuk-link" %></p>
   </div>
-</header>
-<% if @csv_preview %>
-  <%= render(partial: 'truncated_message', locals: { csv_preview: @csv_preview, attachment: @attachment }) if @csv_preview.truncated? %>
-  <div class="csv-preview">
-    <div class="csv-preview-inner">
-      <table>
-        <thead>
-          <tr>
-            <% @csv_preview.headings.each do |heading| %>
-              <th><%= heading %></th>
-            <% end %>
-          </tr>
-        </thead>
+</div>
 
-        <tbody>
-          <% @csv_preview.each_row do |row| %>
-            <tr>
-              <% row.each do |data| %>
-                <td><%= data %></td>
-              <% end %>
-            </tr>
+<main role="main" id="content">
+  <header class="publication-header">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <%= render "govuk_publishing_components/components/title", {
```

Comment:
👋🏼 Stumbled across this Whitehall page as part my heading update:
I'm thinking the the [Inverse Header](https://components.publishing.service.gov.uk/component-guide/inverse_header) component might be more apt here, however I don't think this "Download CSV" secondary link exists  on the current component iteration so the gem might need to be extended  (also, the font looks off on the gem page atm..)

Reply:
I don't know if we'd necessarily need to add a link option like that. It's both very specific and mostly very basic markup (just a paragraph with a link inside and some extra styling for the situation). I can't think of any other places where this would be used. The inverse header though I will deffo sort out, good spot!